src/material/grid-list/grid-tile.ts

```diff
@@ -28,6 +28,10 @@ import {MAT_GRID_LIST, MatGridListBase} from './grid-list-base';
   exportAs: 'matGridTile',
   host: {
     'class': 'mat-grid-tile',
+    // Ensures that the "rowspan" and "colspan" input value is reflected in
+    // the DOM. This is needed for the grid-tile harness.
+    '[attr.rowspan]': 'rowspan',
+    '[attr.colspan]': 'colspan'
```

Comment:
I'm still not sure if it's right that we need to update the component in favor of a test harness. Though our discussion showed that it will be very unreliable if we do the measurements our own (and complex too if we want to respect colspan; rowspan; etc.)

Worth mentioning: If people set the inputs through an attribute, the attribute remains on the element too. This just makes it also show up on the element if the input is set through a binding.

Reply:
I'm more wondering about the use-case for this. How do you see someone writing a test that needs this information?

Reply:
We need it for the `TileCoordinator` that is used to compute the positions for tiles. I don't really know of a good/reasonable test that relies on this information. I figured we just expose it in the harness if we have access to it anyway.

Reply:
That's what I mean- I don't know what kind of test you even would author that would be based on the tile position instead of the tile content. Based on that, it may be simpler to just omit the positional stuff in the harness

Reply:
I was just talking about `colspan` and `rowspan` itself. I think getting a tile by position can be useful since grid tiles might change dynamically, and developers might want to assert that they are positioned as expected.

In general I don't see anything useful one would test in a grid-list. If only content is being tested, then a harness is most likely unnecessary since they can just query inside individual tiles on their own.